### ISSUE TITLE
regulary pull for conversation updates

### ISSUE THREAD

**Issue by mahibi** [nextcloud/talk-android#1591]
### Actual behaviour
When having the conversation list open, there is no regular check for new messages in conversations.
If notifications are working reliable, it doesn't matter that much because the user will be informed by popup (Although the UI of the conversation list is not updated then yet!).
If notifications aren't working then the user might be waiting for updates while having the conversation list open -> But the list won't be updated! (Updates are only pulled when entering the conversation list.)

### Expected behaviour
- regular pull for new messages in conversations list and update UI
- update UI when receiving a notification?

**Comment by mahibi** [comment]
@Ivansss whats the behaviour in talk iOS?

**Comment by nickvergessen** [comment]
It only does when looking at the conversation list. Updating every 30s I think

**Comment by ASerbinski** [comment]
Maybe connect to notify-push if available when the UI is running as an alternative to polling?

**Comment by nickvergessen** [comment]
No, that requires a websocket connection and will drain the battery even more

**Comment by ASerbinski** [comment]
Yes it requires a websocket connection, but no it would not drain the battery more, it would drain the battery a lot less and would also deliver updates instantly rather than having to wait until the next polling interval.
